pkg/neonvm/controllers/webhook.go

```diff
@@ -135,6 +137,16 @@ var _ webhook.CustomValidator = (*VMWebhook)(nil)
 // ValidateCreate implements webhook.CustomValidator
 func (w *VMMigrationWebhook) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
 	vmm := obj.(*vmv1.VirtualMachineMigration)
+	var vm vmv1.VirtualMachine
+	err := w.Client.Get(ctx, types.NamespacedName{Name: vmm.Spec.VmName, Namespace: vmm.Namespace}, &vm)
+	if err != nil {
+		return nil, fmt.Errorf("vm %s not found: %w", vmm.Spec.VmName, err)
```

Comment:
question(blocking): This can fail for other reasons than "not found", right?

Reply:
Well... even if API call failed, we VM is still kind of "not found" 😄 

Not sure if this is the right logic.

Reply:
I guess this is obsolete now, but for the future: There's a handful of unrelated reasons that a request can fail -- e.g., auth, API server restarted, etcd timeout :sweat_smile:

Reply:
Right, "vm not found" usually perceived as "vm does not exist", even though _technically_ it can mean "etcd timouted, so we couldn't find the vm, ergo it is _not found_". 

But I agree it would be confusing.